src/perf_client/scenario_perf_client.cpp

```diff
@@ -130,6 +130,9 @@ class ScenarioPerfClient : public Base
 
 int main(int argc, char** argv)
 {
+  logger::config::loggers().emplace_back(
+    std::make_unique<logger::TextConsoleLogger>());
```

Comment:
Could we have a helper `logger::config::easy_init()` or something similar for a simpler API to be used in tests?

Reply:
Sure, I'll add that.